portal/templates/redesign/register_popup.html

```diff
@@ -0,0 +1,72 @@
+{% extends 'redesign/base_popup.html' %}
+{% load staticfiles %}
+{% load app_tags %}
+
+{% block css %}
+{{ block.super }}
+{% endblock css %}
+
+{% block scripts %}
+{{ block.super }}
+{% endblock scripts %}
+
+{% block pageID %}id='homeWrapper'{% endblock %}
+
+{% block content %}
+{{ block.super }}
+
+<div class="row popup is-flex">
+    <div class="form--register form--register--teacher col-sm-6">
+        <div class="form col-sm-8 col-sm-offset-2">
+            <h1>Register as a teacher</h1>
+            <form id="register-form">
+                <p class="body-text">Once you&rsquo;ve registered your personal details and logged in, you&rsquo;ll be able to create your school or club, or join other teachers at your institution.<br/></p>
+                <p class="small-text">Title</p>
```

Comment:
small-text.  Hm.  If it's genuinely less important, you can use a small tag.  That indicates to search engines and stuff that this text is less important than other text.

Otherwise: what sort of text is this, why is it small?